### DISCUSSION TITLE
django integration: fitering transactions by url

### DISCUSSION THREAD

**Discussion by etienne-lebrun** [getsentry/sentry-python#1569] [Q&A] [answered]
### Problem Statement

I'm trying to reduce the noise in our transactions, by filtering out frequently reached but not interesting urls ( /health in my particular case) 

### Solution Brainstorm

I tried to use the `trace_sampler` function, but at the time it is called, the transaction has the generic name `generic WSGI request` , and the name is only overwritten later to contain the url.
I did not see any opportunity to drop the transaction after the first decision is made.
Am I right that there is currently no way to do such a filtering ?
Thanks!

**Comment by sl0thentr0py** [comment, marked as answer]
hi @etienne-lebrun, for WSGI we include the [`wsgi_environ`](https://github.com/getsentry/sentry-python/blob/b3bd629bc6163d371a45f64fcab37851746efdb7/sentry_sdk/integrations/wsgi.py#L133) object inside `sampling_context`, so you can access the underlying request fields. Something like the following should work.

```python
IGNORED_URLS = ["/ignore/this/url"]

def traces_sampler(sampling_context):
    path = sampling_context.get("wsgi_environ", {}).get("PATH_INFO", None)
    if path and path in IGNORED_URLS:
        return 0
    else:
        return 1
```

**Reply by etienne-lebrun** [reply]
thank you very much, it looks like this will indeed work
